README.md

```diff
@@ -64,7 +64,7 @@ Note `XliffTasks` does not force the items into a sorted order if they are not a
 
 ## Contact
 
-For more information, contact @tmeschter or @nguerrera on GitHub, or file an issue.
+For more information, contact @dotnet/dnceng on GitHub, or file an issue.
```

Comment:
for this link to work it needs to be `@dnceng`.  Let's wait til the afternoon though to make any such changes please.

Reply:
OK Mark confirmed, but please make the change so the at-tag works.

Reply:
I don't understand this--I always see and have to type `@dotnet/dnceng`, `@dnceng` doesn't seem to do anything. (And either way, it's not a link in GitHub Markdown.) Is there a some other context I'm not thinking of?

Reply:
You're right, my mistake.  It doesn't render right in the "view" of the PR.

Reply:
It looks like github flavored markdown uses `@` for emails, which is why they showed up highlighted originally... but yeah, doesn't mean anything here with nothing before the `@`.